Merge Elevator Fixes

cbgpenguin opened the pull request:
Proposed changes to the elevator subsystem

cbgpenguin committed:
Set Height Safety

Abraham17-K commented:
Looks fine to me, though you can just use Math.clamp(min, max, value)
https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/Math.html

cbgpenguin committed:
Clamp and trying to reset encoders

cbgpenguin committed:
Warnings

Added warnings for elevator

cbgpenguin committed:
Slight fixes

Very small changes. READY FOR REVIEW

cbgpenguin committed:
Motion magic? Safety limits??

 Did things, hurt brain. I hope at least something we did works because its all definitely broken now when it would have worked before. Cried trying to convert a voltage to a double I think were tired out from midterms.

cbgpenguin committed:
Main changes

Abraham17-K committed:
Fixed typos

cbgpenguin committed:
Elevator sone things

cbgpenguin committed:
Merge branch 'elevator-fixes' of https://github.com/MontclairRobotics/Reefscape into elevator-fixes
